app/views/spree/shared/_payment.html.haml

```diff
@@ -13,3 +13,9 @@
     = t :email_payment_summary
 - if @order.payments.any?
   = render partial: 'spree/shared/payments_list', locals: { payments: @order.payments }
+  - completedPayments = @order.payments.select {|p| p.state == "checkout" || p.state == "completed"}
+  - if (completedPayments.any?)
+    %p.callout{style: "margin-top: 40px"}
+      %strong
+        = t :email_payment_description
+    %p{style: "margin: 5px"}= payment_method(completedPayments.sort_by(&:created_at).first).description
```

Comment:
again, this line is dangerously assuming too many things and so it's very likely that it'll break in the future.